lib/src/main/scala/spinal/lib/bus/amba4/axi/sim/AxiMemorySim.scala

```diff
@@ -205,6 +205,13 @@ case class SparseMemory() {
     println(s"Loading 0x${byteArray.length.toHexString} bytes from ${file} to 0x${address.toHexString}")
   }
 
+  def loadHex(offset: Int, file: String) : Unit = {
+    import spinal.lib.misc.HexTools.readHexFile
+    readHexFile(file, offset, (address, data) => {
+      writeBigInt(address, data,2)
```

Comment:
```suggestion
      writeBigInt(address, data, 2)
```

Btw, wondering what this 2 stands for.